main/streams/plain_wrapper.c

```diff
@@ -577,9 +577,10 @@ static int php_stdiop_stat(php_stream *stream, php_stream_statbuf *ssb)
 	php_stdio_stream_data *data = (php_stdio_stream_data*) stream->abstract;
 
 	assert(data != NULL);
+	if(ret = do_fstat(data, 1) == 0) {
```

Comment:
Please put parentheses here, playing with C operator priorities is never a good idea.